platform/podjs/src/browserPod.ts

```diff
@@ -399,12 +398,19 @@ class BrowserNetwork {
 
 interface EndpointInfo {
     url: string;
-    auth: string;
+    auth: null; // TODO: change this to an actual type when it's needed. It's null now on endpoints.json
```

Comment:
as now ts can actually read the json, it complains as "auth" there is set both times as null, we might need to update it there to an empty string or leave it as a null here if we're fine with it.

Reply:
But it isn't null here right?
`polyPediaReport: {
        url: process.env.POLYPOD_POLYPEDIA_REPORT_URL,
        auth: process.env.POLYPOD_POLYPEDIA_REPORT_AUTHORIZATION,
    },`